salt/metalk8s/kubeadm/init/kubeconfig/kubelet.sls

```diff
@@ -2,10 +2,23 @@
 
 {%- set hostname = salt.network.get_hostname() %}
 
+{%- set ca_cert = salt['mine.get']('*', 'kubernetes_ca_server').values()[0] %}
+{%- set ca_cert_b64 = salt['hashutil.base64_b64decode'](ca_cert) %}
+
 {%- set cert_info = {'CN': 'system:node:' ~ hostname, 'O': 'system:nodes'} %}
 
 {{ kubeconfig('kubelet', cert_info) }}
 
+Ensure CA cert is present:
```

Comment:
To be done later probably: put this into a dedicated state and call it when necessary because it could be used elsewhere.